src/components/CartButton.jsx

```diff
@@ -48,7 +48,7 @@ const CartButton = () => {
 
     >
         <Button
-            className="blue"
```

Comment:
"Background and foreground colors do not have a sufficient contrast ratio."
![image](https://user-images.githubusercontent.com/49411370/187914602-d620a53a-f9e9-4f93-947a-bbb203be3d72.png)
"These are opportunities to improve the legibility of your content."